docs/transitioning.md

```diff
@@ -230,15 +230,45 @@ This also can be achieved using an operation file similar to the following:
 ## Log Forwarding
 
 Some `_ctl` scripts forward the output and error streams from their server to
-the `logger` program which sends those logs to syslog. BPM does not forward
-logs and so you should use something like the [syslog release][syslog-release]
-to send your logs from the machine to an external log aggregator and store.
+the `logger` program which sends those logs to syslog. 
+
+From `diego-release/jobs/route_emitter/templates/route_emitter_as_vcap.erb`:
```

Comment:
Can we make this more generic, rather than using a specific job from diego-release? Maybe something like `job` rather than `route_emitter`?